### ISSUE TITLE
Add-On does not start

### ISSUE THREAD

**Issue by fionnlagh** [asciidisco/plugin.video.netflix#405]
*I'm submitting a ...*
  - [ ] bug report
  - [ ] feature request
  - [x] support request

### Prerequisites

* [x] Are you running the latest version?
yes. I've installed Kodi nightly via the Ubuntu PPA and I've installed this add-on by using the repository

### Description

I am trying to install the Netflix-Add-On for Kodi. I am running Ubuntu 18.04 LTS and I've installed the Kodi 18 nightly build via the PPA ( ppa:team-xbmc/xbmc-nightly )
First I got an error and according to the debug log, python was missing the cryptodome module. After installing it (and rebooting) I was at least able to enter my username and password. But after entering it, it also threw an error (not in the log, not reproducible for me)
Now everytime I run the add-on it directly throws an error. (Error 111 Connection refused in the debug log)

### Steps to Reproduce

1. Install addon and dependencies
2. run the add-on

**Expected behavior:** [What you expected to happen]
Prompt for credentials and then successfully run the add-on

**Actual behavior:** [What actually happened]
After starting the add-on, an error message pops up, no netflix for me

### Installation

* [ ] I installed the plugin via zip from the Releases page
* [x] I´m using the Netflix Repo
* [ ] I´m using a different source (Please tell which)

### Operating System

* [x] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [ ] Linux (ARM)
* [ ] Android

#### Additional informatin on the environment

I am using Ubuntu 18.04 LTS, Kodi Nightly is installed via PPA

## Debug log

https://paste.ubuntu.com/p/zjPV9Vg9HM/

**Comment by knappster1** [comment]
@fionnlagh I am not sure that this is your issue specifically, but Netflix changed the spelling of something on their website which appears to have made 0.13.8 and older stop working.  I would suggest starting with updating to the latest release, which is now 0.13.10:
https://github.com/asciidisco/plugin.video.netflix/archive/0.13.10.zip

Also, if you're interested in more information on the wrong credentials issues others have had, see:
https://github.com/asciidisco/plugin.video.netflix/issues/402
https://github.com/asciidisco/plugin.video.netflix/issues/398

**Comment by fionnlagh** [comment]
@knappster1 thank you for your reply. 
I have removed the old version of the addon and then installed the version 0.3.10 you have linked.
Unfortunately the add-on still does not start (In debug log:  Error 111 Connection refused) 
Full debug log: https://paste.ubuntu.com/p/WNRHVtz2bh/

**Comment by fionnlagh** [comment]
Okay, that's quite funny now.
I had installed the cryptodome module via apt-get install python-pycryptodome, not via pip2. That was not enough, I also had to install python-pip. Installing this also installed several other python modules as dependencies:
libexpat1-dev libpython-all-dev libpython-dev libpython2.7-dev python-all
  python-all-dev python-asn1crypto python-cffi-backend python-crypto
  python-cryptography python-dbus python-dev python-enum34 python-gi
  python-idna python-ipaddress python-keyring python-keyrings.alt python-pip
  python-pip-whl python-pkg-resources python-secretstorage python-setuptools
  python-six python-wheel python-xdg python2.7-dev
obviously they are needed aswell.

Now I was at least able to login and browse around, but playing any movies does not work (I will maybe create another issue when I can't fix this)
thanks @knappster1 for your help
